src/Integration.Vsix.UnitTests/SonarLintTagger/AccumulatingIssueConsumerTests.cs

```diff
@@ -148,7 +148,7 @@ private class OnIssuesChangedCallbackSpy
         {
             public int CallCount { get; private set; }
             public IList<IAnalysisIssueVisualization> LastSuppliedIssueVisualizations { get; private set; }
-            public IList<IAnalysisIssue> LastSuppliedIssues
+            public IList<IAnalysisIssueBase> LastSuppliedIssues
```

Comment:
The result is taken from LastSuppliedIssueVisualizations, and IssueViz now returns IAnalysisIssueBase